pkgs/development/python-modules/mpv-jsonipc/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchFromGitHub, requests
+, tqdm, websocket_client, pythonOlder }:
+
+buildPythonPackage rec {
+  pname = "mpv-jsonipc";
```

Comment:
this should match the pypi name https://pypi.org/project/python-mpv-jsonipc/
```suggestion
  pname = "python-mpv-jsonipc";
```

Reply:
I didn't realize it was being published on pypi. They're not tagging releases on GitHub though. I'm going to file an issue so we can ship a tagged release instead.

Reply:
https://github.com/iwalton3/python-mpv-jsonipc/issues/1

Reply:
@jonringer: I've got the pname change staged here locally, do I need to also rename the file?

Reply:
yes please